### ISSUE TITLE
PASSWORD RESET: Rephrase of status message when password reset completed

### ISSUE THREAD

**Issue by ovesemart** [SUNET/eduid-front#707]
#### Description of issue:
Just a suggestion...

I would rephrase the klart/success message when password reset is finished successfully to something like:
Lösenordet har uppdaterats or Password is updated

In the English version of the text I would change "Password reset has been updated" to just "Password has been updated" or similar.
--- 

<img width="1170" alt="Skärmavbild 2021-09-14 kl  09 20 06" src="https://user-images.githubusercontent.com/36661180/133385086-00a53446-86f5-4c90-b16b-a4bcf9a3e94e.png">
<img width="1261" alt="Skärmavbild 2021-09-14 kl  09 20 17" src="https://user-images.githubusercontent.com/36661180/133385091-98311f23-bda0-4285-91c7-cbe53568c42f.png">

**Comment by eunjuhuss** [comment]
I think that it will be repetitive to have the same text in the "notification bar" as in the content,
I agree that the English text will need to be updated
